server/services/nl.py

```diff
@@ -17,7 +17,10 @@
 from sentence_transformers import SentenceTransformer
 from sentence_transformers.util import semantic_search
 
+from lib.nl_training import NLQueryClassificationData, NLQueryClassificationModel, NLQueryClassificationType
```

Comment:
nit: I don't think NLQueryClassificationType gets used in this file.

Reply:
Removed. Thanks!